modules/apps/product-navigation/product-navigation-taglib/src/main/resources/META-INF/resources/personal_menu/page.jsp

```diff
@@ -45,98 +45,21 @@ String label = (String)request.getAttribute("liferay-product-navigation:personal
 	<button aria-expanded="true" aria-haspopup="true" class="btn btn-unstyled dropdown-toggle" id="<%= namespace + "personal_menu_dropdown_toggle" %>" ref="triggerButton" type="button">
 		<%= label %>
 	</button>
-</div>
 
-<%
-ResourceURL resourceURL = PortletURLFactoryUtil.create(request, PersonalMenuPortletKeys.PERSONAL_MENU, PortletRequest.RESOURCE_PHASE);
-
-resourceURL.setParameter("currentURL", themeDisplay.getURLCurrent());
-resourceURL.setParameter("portletId", themeDisplay.getPpid());
-resourceURL.setResourceID("/get_personal_menu_items");
-%>
-
-<aui:script require="clay-dropdown/src/ClayDropdown as ClayDropdown,metal-dom/src/dom as dom, metal-position/src/Position">
-	var toggle = document.getElementById('<%= namespace + "personal_menu_dropdown_toggle" %>');
-
-	if (toggle) {
-		dom.once(
-			toggle,
-			'click',
-			function(event) {
-				Liferay.Util.fetch(
-					'<%= resourceURL.toString() %>',
-					{
-						method: 'GET'
-					}
-				).then(
-					function(response) {
-						return response.json();
-					}
-				).then(
-					function(personalMenuItems) {
-						var personalMenu = new ClayDropdown.default(
-							{
-								element: '#<%= namespace + "personal_menu_dropdown_toggle" %>',
-								events: {
-									'willAttach': function(event) {
-										if (<%= expanded %>) {
-											this.expanded = true;
-										}
-
-										var toggleButton = this.element.querySelector('button');
+	<%
+	ResourceURL resourceURL = PortletURLFactoryUtil.create(request, PersonalMenuPortletKeys.PERSONAL_MENU, PortletRequest.RESOURCE_PHASE);
 
-										toggleButton.focus();
+	resourceURL.setParameter("currentURL", themeDisplay.getURLCurrent());
+	resourceURL.setParameter("portletId", themeDisplay.getPpid());
+	resourceURL.setResourceID("/get_personal_menu_items");
 
-										var dropdown = this;
+	Map<String, Object> data = new HashMap<>();
 
-										this.refs.dropdown.refs.portal.on(
-											'menuExpanded',
-											function(event) {
-												var Position = metalPositionSrcPosition.default;
+	data.put("itemsURL", resourceURL.toString());
+	%>
 
-												var menu = this.element;
-												var menuRegion = Position.getRegion(menu);
-
-												if (menuRegion.top < 0) {
-													var body = document.querySelector('body');
-													var bodyRegion = Position.getRegion(body);
-
-													menu.style.top = bodyRegion.top + 'px';
-												}
-											}
-										);
-
-										this.refs.dropdown.refs.portal.on(
-											'rendered',
-											function(event) {
-												if (dropdown.expanded) {
-													this.element.classList.add('dropdown-menu-personal-menu');
```

Comment:
I think we'll need to retain this css class on the component for the overflow styling to work correctly.

Reply:
I'll take a look at the styling differences with @matuzalemsteles and @pat270.

We're also missing the compass icon, which I think just needs a change in the model